wsnet/listen.go

```diff
@@ -18,6 +17,8 @@ import (
 	"cdr.dev/coder-cli/coder-sdk"
 )
 
+const OpNotPermittedByPolicy = "port_not_permitted"
```

Comment:
I did a bad job at explaining my thoughts for this... but it'd be nice if we added a new error type instead. Primarily so we could do `xerrors.Is` to check if perms are denied.

I'm hesitant to make this a new networking Op, since this isn't really a networking error... more of a protocol error.

Reply:
Can be removed